lib/.gitignore

```diff
@@ -0,0 +1,4 @@
+# TODO: remove this file when all files have been replaced with typescript
+configParser.js
```

Comment:
This is still needed. On the prepublish step, the transpiled javascript files are being copied into the lib folder.